ci/jenkins/scripts/test-aks.sh

```diff
@@ -30,7 +30,8 @@ Setup and Run integration tests on AKS cluster with Azure VMs.
         [--azure-tenant-id <TenantID>]              Azure Service Principal Tenant ID.
         [--azure-secret <Secret>]                   Azure Service Principal Secret.
         [--azure-location <Location>]               The Azure location where the setup will be deployed. Defaults to West US 2.
-        [--owner <OwnerName>]                       Setup will be prefixed with owner name."
+        [--owner <OwnerName>]                       Setup will be prefixed with owner name.
+        [--agented]                                 Run test with agented VMs."
```

Comment:
This should be --with-agent. 
https://jenkins.antrea-ci.rocks/job/nephe-test-e2e-aks-with-agent-for-pull-request/2/console